### PR TITLE
Fix Confirmations unit tests

### PR DESCRIPTION
**Pull request by tmancey**
fixes https://github.com/brave/brave-browser/issues/3560

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: vendor/bat-native-confirmations/src/confirmations_impl.cc [hunk 1]
```diff
@@ -50,12 +48,26 @@ ConfirmationsImpl::~ConfirmationsImpl() {
   StopPayingOutRedeemedTokens();
 }
 
+void ConfirmationsImpl::Initialize() {
+  BLOG(INFO) << "Initializing Confirmations";
```
Comment (NejcZdovc) [inline vendor/bat-native-confirmations/src/confirmations_impl.cc]: can we move this into vlog

File: vendor/bat-native-confirmations/src/confirmations_impl.cc [hunk 2]
```diff
@@ -50,12 +48,26 @@ ConfirmationsImpl::~ConfirmationsImpl() {
...
+
+  if (is_initialized_) {
+    LOG(WARNING) << "Already initialized";
```
Comment (NejcZdovc) [inline vendor/bat-native-confirmations/src/confirmations_impl.cc]: can this be info?

File: vendor/bat-native-confirmations/src/confirmations_impl.cc (hunk 1 above)
Comment (tmancey) [reply vendor/bat-native-confirmations/src/confirmations_impl.cc]: Sure right now we do not have VLOG in Confirmations but as soon as merged with Core this is planned

File: vendor/bat-native-confirmations/src/confirmations_impl.cc (hunk 2 above)
Comment (tmancey) [reply vendor/bat-native-confirmations/src/confirmations_impl.cc]: 👍 changed

File: vendor/bat-native-confirmations/src/confirmations_impl.cc (hunk 1 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/confirmations_impl.cc]: great

**Pull request merged**